### PR TITLE
Adding hasArgument and hasProperty

### PR DESCRIPTION
**Pull request by MrBretze**
Hello, i simple add to check if Arguments.java has a argument or not

### CODE REVIEW

File: src/main/java/net/minestom/server/instance/block/BlockAlternative.java [hunk 1]
```diff
@@ -41,6 +41,10 @@ public String getProperty(String key) {
         return null;
     }
 
+    public boolean hasProperty(String key) {
+        return Arrays.stream(properties).anyMatch(s -> s.equals(key));
```
Comment (TheMode) [inline src/main/java/net/minestom/server/instance/block/BlockAlternative.java:45]: Should't it be
```java
return getProperty(key) != null;
```
?

Comment (Geolykt) [reply src/main/java/net/minestom/server/instance/block/BlockAlternative.java:45]: either way this implementation has it's issues as it does not match the implementation of getProperty() as well as that using a Stream might be a bit too overkill for that

Comment (MrBretze) [reply src/main/java/net/minestom/server/instance/block/BlockAlternative.java:45]: Yeah, i dont not why i am using a Stream sorry ><

**Comment by TheMode** [comment]
Currently `CommandContext#has`

**Pull request closed without merging**